### DISCUSSION TITLE
Cheering for you, man!

### DISCUSSION THREAD

**Discussion by Talha1788** [namidaco/namida#264] [General]
Hey, man. I just wanted to let you know that you’re doing a great job!

As a music lover, my last players history is 189h+. One of my friend told me about your app and gave me the apk. I've then installed your app and experimented with it. Tasted all the features it had to offer. And I'm very satisfied with the app. Now I've fully shifted to this app. And I've nominated this app as my primary music player. I'm very glad that I got to know about this masterpiece!

Keep up the good work, man! 
We're with you.

**Comment by MSOB7YY** [comment]
omgggg
woah so glad u really enjoyed it and thanks a lottt for telling this! its always refreshing seeing cute ppl like you so im happy for it, and don't hesitate asking for stuff, I'm with y'all too ^^ tysmm
